tests/functional/validation.py

```diff
@@ -0,0 +1,95 @@
+import concurrent.futures
+import csv
+import io
+import os
+import shutil
+import tempfile
+
+import hca.dss
+import loompy
+import numpy
+import requests
+import s3fs
+import zarr
+
+
+DSS_CLIENT = hca.dss.DSSClient()
+DSS_CLIENT.host = f"https://dss.{os.environ['DEPLOYMENT_STAGE']}.data.humancellatlas.org/v1"
+
+S3 = s3fs.S3FileSystem(anon=True)
+
+
+def calculate_ss2_metrics_direct(bundle_fqids):
```

Comment:
Cool! How long does this take for the default 200 bundle test `test_matrix_service_ss2`?

Reply:
The whole set of tests is like 5 minutes.